code/datums/spells/sentient_sword_lunge.dm

```diff
@@ -16,7 +16,14 @@
 		to_chat(user, "<span class='warning'>You cannot use this ability if you're outside a blade!</span>")
 		return
 	var/obj/item/nullrod/scythe/talking/user_sword = user.loc
-	var/mob/living/carbon/holder = user_sword.loc
-	if(istype(holder))
+	if(ishuman(user_sword.loc))
+		var/mob/living/carbon/holder = user_sword.loc
 		holder.unEquip(user_sword)
+	else if(isstorage(user_sword.loc))
+		if(prob(50))
+			to_chat(user, "<span class='warning'>You fail to break out of [user_sword.loc]!</span>")
+			return
+		var/turf/our_turf = get_turf(user_sword.loc)
+		our_turf.visible_message("<span class='danger'>[user_sword] magically teleports out of [user_sword.loc]!</span>")
```

Comment:
```suggestion
		our_turf.visible_message("<span class='danger'>[user_sword] furiously lunges out of [user_sword.loc]!</span>")
```
small suggestion, flows a little better than "teleports out"

Reply:
does bread lunge? Up to what you think

Reply:
replaced with words from discord